packages/ui-webc/src/components/text-area/text-area.tsx

```diff
@@ -0,0 +1,50 @@
+import { Component, h, Prop } from "@stencil/core";
+
+@Component({
+  tag: "scout-text-area",
+  styleUrl: "text-area.css",
+  shadow: {
+    delegatesFocus: true,
+  },
+})
+
+export class ScoutTextArea {
+  @Prop() autofocus: boolean = false;
+
+  @Prop() disabled?: boolean;
+
+  @Prop() form: string;
+
+  @Prop() maxLength?: number;
+
+  @Prop() placeholder?: string;
+
+  @Prop() readOnly?: boolean;
+
+  @Prop() required?: boolean;
+  
+  @Prop() value?: string;
+
+  @Prop() label?: string;
+
+  render() {
+    return (
+      <div class="flexbox">
+      <label>
+        {this.label}
+      </label>
+      <textarea 
```

Comment:
Can we also add the `name` prop? Just realized we're missing that on all inputs. I'll add it to the others in a separate PR 😊